docs/cache/develop/integrations/sequelize.md

```diff
@@ -0,0 +1,75 @@
+---
+sidebar_position: 6
+sidebar_label: Sequelize
+title: Integrate Momento With Sequelize ORM
+description: Supercharge your sequelize backed databases with Momento cache
+---
+
+# Integrating Momento with your Sequelize backed database
+
+[Sequelize](https://sequelize.org/) is a popular ORM for Oracle, Postgres, MySQL, MariaDB, SQLite and SQL Server, and more.
+
+In this tutorial, we'll make use of a simple [example](https://github.com/momentohq/momento-sequelize-cache/tree/main/examples) from our [Momento sequelize model cache wrapper](https://github.com/momentohq/momento-sequelize-cache). By the end of the exercise, you'll be able to seamlessly cache your sequelize-backed ORM databases with minimal code changes.
+
+## Prerequisites
+
+To get this example running, you'll need to have the following available:
+
+- An account with a Git provider such as [GitHub](https://github.com/), [GitLab](https://gitlab.com), or [Bitbucket](https://bitbucket.org/).
+- A copy or fork of the [Momento Sequelize Wrapper](https://github.com/momentohq/momento-sequelize-cache) in your personal repositories.
```

Comment:
why do they need to fork? are we not publishing this to npm.js?

Reply:
Oh I thought examples will not be a part of the npmjs release. I can change this doc to not use examples though and directly integrate with us